src/View/StringTemplate.php

```diff
@@ -231,6 +231,11 @@ public function format($name, array $data)
         if ($template === null) {
             return '';
         }
+
+        if (isset($data['templateParams'])) {
```

Comment:
I used the name suggested in the original issue, but I too found myself typing `templateVars` at times so will change :)